starter-code/server.js

```diff
@@ -146,7 +146,7 @@ app.listen(PORT, function() {
 ////////////////////////////////////////
 function loadArticles() {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // (3), (4), (5), R of CRUD, not sure on which method...
```

Comment:
another function is instantiating this one. Also this is tricky but Retrieve is not the CRUD operation in this case.